### PR TITLE
Seperating every port and starboard hallway areas into three areas each (fore midship aft)

### PR DESCRIPTION
**Pull request by Huffie56**
# About the pull request
1-Main idea is to split each Hallway area port and starboard into three areas.(fore midship aft)
2-Clean up the map to remove emergency shutter that are not acting as a frontier between areas.
3-removed unused areas.
4-rebuild structure of hallway area to have /lower and /upper areas.

# Explain why it's good for the game
1-Mainly good to allow better use of health monitor.
2-trying to standardize how emergency shutter are placed on the almayer
3-4 are refactor with no impact on player.
# Testing Photographs and Procedure
<details>
<summary>Screenshots & Videos</summary>

Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.

</details>

# Changelog
:cl:
maptweak: split each Hallway area port and starboard into three areas.(fore midship aft)
maptweak: change the area around north and south west staircase to remove emergency shutters.
maptweak: change the area west of squad preparation room to remove emergency shutter.
maptweak: change the area west of lower engi to make emergency shutter be at the frontier of the area.
/:cl:

### CODE REVIEW

**Comment by cm13-github** [comment]
This pull request has conflicts, please resolve those before we can evaluate the pull request.

File: code/game/area/almayer.dm
Comment (Nanu308) [inline code/game/area/almayer.dm:1]: bilde
Naming Convention, "Port-Fore / Starboard-Fore"

**Review by Nanu308 — CHANGES_REQUESTED** [review]

File: code/game/area/almayer.dm
Comment (Huffie56) [reply code/game/area/almayer.dm:1]: done

**Review by Nanu308 — APPROVED** [review]
seems fine now

**Pull request merged**